lib/env/commands/remove_default.rb

```diff
@@ -0,0 +1,40 @@
+# =============================================================================
+# Copyright (C) 2022-present Alces Flight Ltd.
+#
+# This file is part of Flight Environment.
+#
+# This program and the accompanying materials are made available under
+# the terms of the Eclipse Public License 2.0 which is available at
+# <https://www.eclipse.org/legal/epl-2.0>, or alternative license
+# terms made available by Alces Flight Ltd - please direct inquiries
+# about licensing to licensing@alces-flight.com.
+#
+# Flight Environment is distributed in the hope that it will be useful, but
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, EITHER EXPRESS OR
+# IMPLIED INCLUDING, WITHOUT LIMITATION, ANY WARRANTIES OR CONDITIONS
+# OF TITLE, NON-INFRINGEMENT, MERCHANTABILITY OR FITNESS FOR A
+# PARTICULAR PURPOSE. See the Eclipse Public License 2.0 for more
+# details.
+#
+# You should have received a copy of the Eclipse Public License 2.0
+# along with Flight Environment. If not, see:
+#
+#  https://opensource.org/licenses/EPL-2.0
+#
+# For more information on Flight Environment, please visit:
+# https://github.com/openflighthpc/flight-env
+# ==============================================================================
+module Env
+  module Commands
+    class RemoveDefault < Command
+      def run
+        if @args.any?
+          raise OptionParser::InvalidArgument, "do not specify environment when removing default"
+        end
```

Comment:
Is this possible to exercise?  I would have expected command to complain about additional arguments.  Perhaps, it doesn't do that because we have an old command in use for env?